Assets/StreamingAssets/WorldGen/DefaultStartArea.sav

```diff
@@ -2720,6 +2720,5 @@
       "Type": "power_cable"
     }
   ],
-  "RoomBehaviors": []
-  }
+  "RoomBehaviors": [],
```

Comment:
I accidentally left that in from testing if setting start time in start area save file would work (it does)... but surprisingly doesn't cause a parsing error.

Regardless, I'll see about removing it next time I'm editing the code.